crates/polars-core/src/frame/mod.rs

```diff
@@ -411,6 +411,29 @@ impl DataFrame {
         DataFrame { columns }
     }
 
+    /// Create a new `DataFrame` but does not check the length of the `Series`,
+    /// only check for duplicates.
+    ///
+    /// It is advised to use [Series::new](Series::new) in favor of this method.
```

Comment:
I think this should reference DataFrame?

```suggestion
    /// It is advised to use [DataFrame::new](DataFrame::new) in favor of this method.
```